### ISSUE TITLE
Minimum Pygments version

### ISSUE THREAD

**Issue by whiteinge** [amjith/pgcli#46]
Cent 6 has a very outdated python-pygments package. (It was already installed on my system as a dep for an unrelated package.) I didn't dive in to the traceback to see if Pygments changed something or if SQL support is new (would be weird but maybe?). In any case the traceback is below. Upgrading Pygments (installed via pip) fixed the problem. Perhaps a pre-flight version check is needed?

```
$ pgcli
Traceback (most recent call last):
  File "/usr/bin/pgcli", line 9, in <module>
    load_entry_point('pgcli==0.10.0', 'console_scripts', 'pgcli')()
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 299, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 2229, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 1948, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/usr/lib/python2.6/site-packages/pgcli/main.py", line 16, in <module>
    from pygments.lexers.sql import SqlLexer
ImportError: No module named sql
```

**Comment by amjith** [comment]
Yes! I'm adding a version check to setup.py to make sure Pygments and other libs are recent enough.

**Comment by amjith** [comment]
Fixed by commit d437a12a03f7afce97340c11084139c8a08bb6eb.

This will go out in next release. 

Thank you for the report. :rocket:

**Comment by whiteinge** [comment]
Thanks for the quick fix! :-)
